openedx/core/djangoapps/course_apps/rest_api/v1/views.py

```diff
@@ -185,6 +194,13 @@ def patch(self, request: Request, course_id: str):
             course_app = None
         if not course_app or not course_app.is_available(course_key):
             raise ValidationError({"id": "Invalid app ID"})
-        set_course_app_enabled(course_key=course_key, app_id=app_id, enabled=enabled, user=request.user)
-        serializer = CourseAppSerializer(course_app, context={"course_key": course_key, "user": request.user})
+        is_enabled = set_course_app_enabled(course_key=course_key, app_id=app_id, enabled=enabled, user=request.user)
+        serializer = CourseAppSerializer(
+            course_app,
+            context={
+                "course_key": course_key,
+                "user": request.user,
+                "app_status": {app_id: is_enabled},
+            },
```

Comment:
should we test these keys as well?

Reply:
Not sure I follow. Test these in what way? i.e. test the serializer?

Reply:
@xitij2000 probably I missed something, I just wanted to see how are we testing the new key that we added i.e app_status, here.